scripts/download-oas.sh

```diff
@@ -38,6 +38,25 @@ git clone ${OAS_REPO}
 for service_dir in ${work_dir}/${OAS_REPO_NAME}/services/*; do
     max_version_dir=""
     max_version=-1
+    service=$(basename "$service_dir")
+
+    if [[ ${service} == "iaas" ]]; then
+        for dir in ${service_dir}/*; do
+            version=$(basename "$dir")
+            echo "found iaas with version ${version}"
+            # Check if directory name starts with 'v'
+            if [[ ${version} == v* ]]; then
+                # Remove the 'v' prefix
+                version=${version#v}
+                # Check if version is alpha
+                if [[ ${version} == *alpha* ]]; then 
+                    echo "found alpha iaas"
+                    mv -f ${dir}/*.json ${ROOT_DIR}/oas/iaasalpha.json
+                    continue 
```

Comment:
Wouldn't be enough to add an
```bash
if [[ ${service} == "iaas" ]]; then
    echo "found alpha iaas"
    mv -f ${dir}/*.json ${ROOT_DIR}/oas/iaasalpha.json
fi
```
inside the `if [[ ${version} == *alpha* ]]; then` below?

Reply:
Because the rest of the logic is similar, right?

Reply:
yeah true, that's much simpler